resources/lang/en/thoughts/alive/general.json

```
@@ -4412,7 +4416,7 @@
     {
         "id": "gen_to_dead_parent",
         "thoughts": [
-            "Heard r_c humming a tune in {PRONOUN/m_c/poss} dream that {PRONOUN/m_c/subject} remembered r_c humming in the nursery",
+            "Heard a humming tune in {PRONOUN/m_c/poss} dream that {PRONOUN/m_c/subject} remembered r_c humming in the nursery",
```

```suggestion
            "Heard a tune in {PRONOUN/m_c/poss} dream that {PRONOUN/m_c/subject} remembered r_c humming in the nursery",
```
Better not to repeat ourselves